apps/web/src/routes/_content/resources/books/$bookId.tsx

```diff
@@ -76,58 +88,112 @@ function Book() {
+                      {t('words.publicationDate')}
                     </span>
-                  ))}
+                    <span className="body-14px md:body-16px text-white ">
+                      {book?.publicationYear}
+                    </span>
+                  </div>
                 </div>
+
                 {isScreenMd && displayAbstract()}
               </div>
-            </div>
+            </article>
             {!isScreenMd && displayAbstract()}
           </Card>
-
-          <div className="mx-auto my-6 flex flex-row justify-between p-2">
-            <div className="flex flex-col">
-              {!book?.summaryText && (
-                <BookSummary
-                  content={book?.summaryText as string}
-                  title={book?.title ? book?.title : ''}
-                />
-              )}
-            </div>
-          </div>
         </div>
       )}
+      <section className="mt-8 lg:mt-[100px]">
+        <h3 className="label-medium-med-16px md:title-large-24px font-medium leading-none md:leading-[116%] text-white mb-5 md:mb-10">
+          {t('resources.pageSubtitleBooks')}
+        </h3>
+        <Carousel>
+          <CarouselContent>
+            {isFetchedSuggestedBooks ? (
+              suggestedBooks?.map((suggestedBook) => {
+                const isBook =
+                  'cover' in suggestedBook && 'title' in suggestedBook;
+                if (isBook) {
+                  return (
+                    <CarouselItem
+                      key={suggestedBook.id}
+                      className="text-white basis-1/2 md:basis-1/2 lg:basis-1/4 relative w-full bg-gradient-to-r max-w-[282px] max-h-[350px] rounded-2xl lg:rounded-[22px]"
+                    >
+                      <Link to={`/resources/books/${suggestedBook.id}`}>
+                        <div className="relative h-full">
+                          <img
+                            className="max-h-72 sm:max-h-96 size-full object-cover rounded-[10px]"
+                            alt={suggestedBook.title}
+                            src={suggestedBook.cover}
+                          />
+                          <div
+                            className="absolute inset-0 -bottom-px rounded-[10px]"
+                            style={{
+                              background: `linear-gradient(360deg, rgba(40, 33, 33, 0.90) 10%, rgba(0, 0, 0, 0.00) 60%),
+                              linear-gradient(0deg, rgba(57, 53, 49, 0.20) 0%, rgba(57, 53, 49, 0.20) 100%)`,
+                              backgroundSize: '153.647% 100%',
+                              backgroundPosition: '-5.216px 0px',
+                              backgroundRepeat: 'no-repeat',
+                            }}
+                          />
+                        </div>
+
+                        <h3 className="absolute px-2 lg:px-4 body-14px lg:title-large-24px pb-1 lg:pb-5 bottom-px line-clamp-2">
```

Comment:
use margin instead of padding bottom to prevent this issue
![image](https://github.com/user-attachments/assets/bbc9e1b7-1a01-4a2d-9603-d7d212656f05)